packages/core-mobile/src/utils/screenOptions.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import {
+  StackCardInterpolatedStyle,
+  StackCardInterpolationProps,
+  StackNavigationOptions,
+  TransitionPresets
+} from '@react-navigation/stack'
+import { Animated } from 'react-native'
+import Grabber from 'components/navigation/Grabber'
+import BackBarButton from 'components/navigation/BackBarButton'
+
+export const stackNavigatorScreenOptions: StackNavigationOptions = {
```

Comment:
nit: can you add comments for some of these consts to make it clearer when we should use them?